src/features/product/productSlice.ts

```diff
@@ -0,0 +1,300 @@
+import {createSlice} from '@reduxjs/toolkit';
+import syltherine from "@/assets/furniture/syltherine.png";
+import liviosa from "@/assets/furniture/leviosa.png";
+import lolito from "@/assets/furniture/lolito.png";
+import respira from "@/assets/furniture/respira.png";
+import grifo from "@/assets/furniture/grifo.png";
+import muggo from "@/assets/furniture/muggo.png";
+import pingky from "@/assets/furniture/pingky.png";
+import potty from "@/assets/furniture/potty.png";
+
+export type ProductType = {
+    id: number
+    src: string
+    alt: string
+    title: string
+    description: string
+    price: string
+    oldPrice: string
+    status: string
+    quantity: number
+}
+
+const initialState: ProductType[] = [
+    {
+        id: 1,
+        src: syltherine,
+        alt: 'syltherine',
+        title: 'Syltherine',
+        description: 'Stylish cafe chair',
+        price: '2500000',
+        oldPrice: '3.500.000',
```

Comment:
why do you use strings here? I would expect that `price` and `oldPrice` would have `number` type